evalml/tests/automl_tests/test_automl_search_classification.py

```diff
@@ -277,30 +276,10 @@ def test_non_optimizable_threshold(mock_fit, mock_score, X_y_binary):
     automl.search()
     mock_fit.assert_called()
     mock_score.assert_called()
-    assert automl.best_pipeline.threshold is not None
-    assert automl.results['pipeline_results'][0]['cv_data'][0].get('binary_classification_threshold') == 0.5
-    assert automl.results['pipeline_results'][0]['cv_data'][1].get('binary_classification_threshold') == 0.5
-    assert automl.results['pipeline_results'][0]['cv_data'][2].get('binary_classification_threshold') == 0.5
-
-
-@patch('evalml.pipelines.MulticlassClassificationPipeline.score')
-@patch('evalml.pipelines.MulticlassClassificationPipeline.fit')
-def test_non_optimizable_threshold_multi(mock_fit, mock_score, X_y_multi):
```

Comment:
This test doesn't belong in the `AutoMLSearch` tests! No thresholds are defined for multiclass pipelines.